deflate.c

```diff
@@ -590,7 +590,7 @@ int32_t ZEXPORT PREFIX(deflatePrime)(PREFIX3(stream) *strm, int32_t bits, int32_
         s->sym_buf < s->pending_out + ((BIT_BUF_SIZE + 7) >> 3))
         return Z_BUF_ERROR;
     do {
-        put = BIT_BUF_SIZE - s->bi_valid;
+        put = (BIT_BUF_SIZE-1) - s->bi_valid;
```

Comment:
This essentially reduces the bit buffer size by one bit, which is a bad thing...

Reply:
Fixed with your changes.